backend/src/test/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapperTest.java

```diff
@@ -285,11 +282,11 @@ void map_WhenReportedSammToProductItemStock_ReturnsMultipleReportedProductItemSt
         mpr.setPartnerBuysMaterial(false);
```

Comment:
This may be an older oversight. 

But the Material that is created in line 273 (and following) is later on used on the Supplier's Side, when he is de-serializing the SAMM that was created on the Customer side. Therefore, in line 272 the "ownMaterialNumber" attribute should be set to SUPPLIER_MAT_NUMBER. 

Accordingly, Line 286 ("setPartnerMaterialNumber") should be set to CUSTOMER_MAT_NUMBER